documentation/book/proc-configuring-kafka-mirror-maker-authentication-tls.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-mirror-maker-authentication.adoc
+
+[id='proc-configuring-kafka-mirror-maker-authentication-tls-{context}']
+= Configuring TLS client authentication in Kafka Mirror Maker
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster
+* A running Cluster Operator with a `tls` listener with `tls` authentication enabled
+
+.Procedure
+
+Because Kafka Mirror Maker connects to two Kafka clusters (source and target), it is possible to configure TLS client authentication for one of them or both.
+Following the steps for configuring TLS client authentication on the consumer side for connecting to the source Kafka cluster.
+
+. Find out the name of the `Secret` with the public and private keys which should be used for TLS Client Authentication and the keys under which they are stored in the `Secret`.
+If such a `Secret` does not exist yet, prepare the keys in a file and create the `Secret`.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl create`:
+[source,shell,subs=+quotes]
+kubectl create secret generic _<my-secret>_ --from-file=_<my-public.crt>_ --from-file=_<my-private.key>_
++
+endif::Kubernetes[]
+On {OpenShiftName} this can be done using `oc create`:
++
+[source,shell,subs=+quotes]
+oc create secret generic _<my-secret>_ --from-file=_<my-public.crt>_ --from-file=_<my-private.key>_
+. Edit the `KafkaMirrorMaker.spec.consumer.authentication` property.
+For example:
++
+[source,yaml,subs=attributes+]
+----
+apiVersion: {KafkaApiVersion}
+kind: KafkaMirrorMaker
+metadata:
+  name: my-mirror-maker
+spec:
+  # ...
+  consumer:
+    authentication:
+      type: tls
+      certificateAndKey:
+        secretName: my-secret
+        certificate: my-public.crt
+        key: my-private.key
+  # ...
+----
++
+. Create or update the resource.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl apply`:
+[source,shell,subs=+quotes]
+kubectl apply -f _<your-file>_
++
+endif::Kubernetes[]
+On {OpenShiftName} this can be done using `oc apply`:
++
+[source,shell,subs=+quotes]
+oc apply -f _<your-file>_
+
+The same steps can be done for configuring TLS client authentication on the producer side for connecting to the target Kafka cluster.
```

Comment:
Rewrite as suggested in the "Configuring SCRAM-SHA-512 authentication in Kafka Mirror Maker" module.